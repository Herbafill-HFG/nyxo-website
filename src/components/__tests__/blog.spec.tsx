import React from "react"
import renderer from "react-test-renderer"

import TagFilter from "../Blog/TagFilter"

describe("TagFilter", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<TagFilter />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
