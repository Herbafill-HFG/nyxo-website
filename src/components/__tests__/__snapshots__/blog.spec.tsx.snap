// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TagFilter renders correctly 1`] = `
<ul
  className="sc-fzokOt fUljbT"
>
  <li
    className="sc-fznZeY fLrtke"
  >
    <a
      className="sc-fznKkj iyxlmx"
      href="/tags"
    >
      All Tags
    </a>
  </li>
</ul>
`;
